doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3407 @@
+{
```

Comment:
<p>NIT: I think this paragraph is too long-winded</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>agree! Improved</p>